### PR TITLE
[SIG-4336] Move summary map

### PR DESCRIPTION
**Pull request by janjaap**
Moves the static map on the summary page from section 1 to section 2.

### CODE REVIEW

File: src/components/IconList/IconList.tsx [hunk 1]
```diff
@@ -37,7 +37,9 @@ export const IconListItem: FunctionComponent<IconListItemProps> = ({
   isReported,
 }) => (
   <StyledListItem data-testid={id} className={className}>
-    <StyledImg alt="" height={iconSize} src={iconUrl} width={iconSize} />
+    {iconUrl && (
+      <StyledImg alt="" height={iconSize} src={iconUrl} width={iconSize} />
+    )}
     {isReported && (
```
Comment (kiruys) [inline src/components/IconList/IconList.tsx:43]: 👍

**Comment by kiruys** [comment]
obviously some work, but perhaps worth it... remove the list naming in assetlist, iconlist, iconlistitem..
what do you think?

File: src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx [hunk 2]
```diff
@@ -52,45 +51,30 @@ const AssetList: FunctionComponent<AssetListProps> = ({
...
-        isReported={item.isReported}
+        key={id}
+        id={isReported ? `assetListItem-${id}-reported` : `assetListItem-${id}`}
+        iconUrl={icon?.iconUrl}
+        isReported={isReported}
       >
         <ItemWrapper>
           <StyledLabel>
-            <div>{item.label}</div>
-            {item.isReported && <StyledDiv>Is gemeld</StyledDiv>}
```
Comment (kiruys) [inline src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx:89]: This div is necessary for the flex layout.

Comment (janjaap) [reply src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx:89]: Don't think so; `StyledLabel` is not a flex element.

Comment (kiruys) [reply src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx:89]: Perhaps you are right, but the styling is definitely messed up if you remove the div

**Review by kiruys — APPROVED** [review]

**Pull request merged**